site/docs/v1/tech/apis/system.adoc

```diff
@@ -92,12 +93,12 @@ include::docs/v1/tech/apis/_system-configuration-response-body.adoc[]
 Available Since Version 1.16.0
 ====
 
-This API is used to export the System Logs, the response will be a compressed zip archive containing the logs from the configured log directory. When running FusionAuth on Docker or other container service where logs are written to `stdout` and not written to the file system, this API will return an empty archive.
+This API is used to export the System Logs, the response will be a compressed zip archive containing the logs from the configured log directory. When running FusionAuth on Docker or other container services where logs are written to `stdout` and not written to the file system, this API will return an empty archive.
```

Comment:
I think `service` (non plural) was intentional here. I suppose you could write it either way, but I think we are comparing it to `Docker` when we say `Docker or other container service` - so we are comparing a `Docker` to another `container service`.